website/content/docs/v0.11/Cloud Platforms/azure.md

```diff
@@ -209,7 +209,7 @@ az vm availability-set create \
   -g $GROUP
 
 # Create the controlplane nodes
-for i in $( seq 1 3 ); do
+for i in $( seq 0 1 2 ); do
```

Comment:
or just `0 2`?

Reply:
yeah, I did 0 1 2 because that's what we already used in the NICs and IPs for some reason.